src/main/java/br/unb/cic/analysis/ioa/InterproceduralOverrideAssignment.java

```diff
@@ -92,27 +117,36 @@ protected void internalTransform(String s, Map<String, String> map) {
      * @param flowChangeTag This parameter identifies whether the unit under analysis is in the flow of any statement already marked.
      *                      Initially it receives the value IN_BETWEEN but changes if the call to the current method (sootMethod) has been marked as SOURCE or SINK.
      *                      The remaining statements of the current method that have no markup will be marked according to the flowChangeTag.
+     * @return the result of applying the analysis considering the income abstraction (in) and the sootMethod
      */
-    private void traverse(SootMethod sootMethod, Statement.Type flowChangeTag) {
-
+    private FlowSet<DataFlowAbstraction> traverse(FlowSet<DataFlowAbstraction> in, SootMethod sootMethod,
+                                                  Statement.Type flowChangeTag) {
+        //System.out.println(sootMethod);
         if (this.traversedMethods.contains(sootMethod) || sootMethod.isPhantom()) {
-            return;
+            return in;
         }
 
         this.traversedMethods.add(sootMethod);
 
-        this.body = retrieveActiveBodySafely(sootMethod);
+        Body body = retrieveActiveBodySafely(sootMethod);
```

Comment:
Não seria this.body mesmo? Já que a classe tem esse atributo?

Reply:
Eu removi o atributo.  Antes era usando em outros métodos, atualmente Body só é usado nesse método então não faz sentido ser um atributo.